scene/3d/sprite_3d.cpp

```diff
@@ -383,22 +383,40 @@ SpriteBase3D::SpriteBase3D() {
 	modulate = Color(1, 1, 1, 1);
 	pending_update = false;
 	opacity = 1.0;
-	immediate = VisualServer::get_singleton()->immediate_create();
-	set_base(immediate);
+
```

Comment:
Do you need to use a material?  I think there were some materials that can be created specially for 3D sprites

Reply:
Yea. Each needs a material because they need to have different textures.

Reply:
I left the material stuff, But I retained the Shader caching so the rebinds are not so bad. We need to have different materials so each Sprite3D can have a unique texture.